recipes/speech_to_text/stt.js

```diff
@@ -55,21 +95,21 @@ tj.listen(function(msg) {
     var containsSet = msg.indexOf("set") >= 0;
     var containsLight = msg.indexOf("the light") >= 0;
     var containsDisco = msg.indexOf("disco") >= 0;
-
-    if ((containsTurn || containsChange || containsSet) && containsLight) {
-        // was there a color uttered?
-        var words = msg.split(" ");
-        for (var i = 0; i < words.length; i++) {
-            var word = words[i];
-            if (colors[word] != undefined || word == "on" || word == "off") {
-                // yes!
-                tj.shine(word);
-                break;
-            }
-        }
-    } else if (containsDisco) {
-        // discoParty();
-    }
+    //
+    // if ((containsTurn || containsChange || containsSet) && containsLight) {
+    //     // was there a color uttered?
+    //     var words = msg.split(" ");
+    //     for (var i = 0; i < words.length; i++) {
+    //         var word = words[i];
+    //         if (colors[word] != undefined || word == "on" || word == "off") {
+    //             // yes!
+    //             tj.shine(word);
+    //             break;
+    //         }
+    //     }
+    // } else if (containsDisco) {
+    //     // discoParty();
+    // }
```

Comment:
commenting this block breaks the functionality of this code -- was this intentional or accidental?